StartupSession/Link/Create.aplf

```diff
@@ -69,7 +69,12 @@ NOHOLD:
                  :EndIf
                  emptyns←1
              :Else
-                 emptyns←0∊⍴U.ListNames nsref←⍎ns
+                 :If 0≠≢names←U.ListNames nsref←⍎ns
+                     m←(((⍕nsref),'.')∘,¨names)∊⎕XSI          ⍝ Name on the stack
```

Comment:
Is `ns` not already `⍕nsref`? (Check if it will always be normalised!) Too many nested parens, and use a meaningful name:
```apl
onStack←(ns,¨'.',¨names)∊⎕XSI
```